README.rst

```diff
@@ -93,6 +93,90 @@ be that station names are not recognized correctly because they contained the
 because of this please download new data from the ISMN since this issue should
 be fixed.
 
+Landcover Classification
```

Comment:
I added the insitu measurements and a short description of how to use it, because as the in-situ values are different from the ESA CCI classification. Same for climate classification.